include/configuration/AvailableArchitecture.hpp

```diff
@@ -35,6 +37,10 @@ enum class AvailableArchitecture {
             return "IBMQ_Casablanca";
         case AvailableArchitecture::IBMQ_Tokyo:
             return "IBMQ_Tokyo";
+        case AvailableArchitecture::Rigetti_Agave:
+            return "Rigetti_Agave";
+        case AvailableArchitecture::Rigetti_Aspen:
+            return "Rigetti_Aspen";
```

Comment:
In order to get the CI checks to pass, you have to add a test that uses these architectures.